lib/python/rose/scheme_handler.py

```diff
@@ -123,5 +124,6 @@ def guess_handler(self, item):
             return handler
         for handler in self.handlers.values():
             can_handle = getattr(handler, self.can_handle, None)
-            if (callable(can_handle) and can_handle(item)):
+            if (isinstance(can_handle,
+                           collections.abc.Callable) and can_handle(item)):
```

Comment:
I have just tried using `callable` in my Python 3.7 environment. It is still there. Is this an automated change by `2to3`? Another approach should be to call `can_handle(item)` any way and catch `TypeError`.

Reply:
'tis automated change. Gah!